inst/components/scss/sidebar.scss

```
@@ -118,30 +120,49 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
     }
   }
 
+  &[data-bslib-sidebar-open="always"] {
+    > .sidebar > .sidebar-content {
+      // remove toggle button padding adjustment if there isn't a button
+      padding-top: var(--bslib-sidebar-padding);
+    }
+  }
```

This way we wouldn't need this logic in two places?

```suggestion
  &:not([data-bslib-sidebar-open="always"]) {
    > .sidebar > .sidebar-content {
      // remove toggle button padding adjustment if there isn't a button
      padding-top: calc(var(--bslib-sidebar-padding) * 2);
    }
  }
```

---

Yeah good point (note to self the change should be higher up closer to where `padding-top` is modified currently)

---

Oh wait, I was wrong, it has to be here because the `data-bslib-sidebar-open` attribute appears on the layout element and the `.sidebar-content` class is nested too deeply above.

I'm torn about taking this change and I am leaning toward not taking it. I generally would prefer to use the positive state rather than the negation, i.e. here we know specifically that the `"always"` sidebars don't want the extra padding. I think this CSS will be easier to maintain in the long run if the rule matches the nature of the exception.